### ISSUE TITLE
#170 'Active Flows - No Results Found' is still exsit in V2.1.151018

### ISSUE THREAD

**Issue by nazhu** [ntop/ntopng#245]
In CentOS 6.7, the issue #170 'Active Flows - No Results Found' is still exsit, but In Debian 8 i386 and x86_64, it's OK.

**Comment by simonemainardi** [comment]
Active flows work properly with the following setup:

```
[mainardi@centos6 ~]$ cat /etc/centos-release 
CentOS release 6.7 (Final)
[mainardi@centos6 ~]$ uname -a
Linux centos6.ntop.org 2.6.32-573.7.1.el6.x86_64 #1 SMP Tue Sep 22 22:00:00 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
[mainardi@centos6 ~]$ ntopng --version
v.2.1.151021 [Professional Edition]
GIT rev:   dev:4b4ef92a6d34ef1d4ecec22d54ba1929989418ed:20151021
Pro rev:   r507
System Id: 12DC4E4000000789
Built on:  CentOS release 6.6 (Final)
[mainardi@centos6 ~]$ 
````

are you aligned with the above configuration?

![image](https://cloud.githubusercontent.com/assets/3397663/10644977/98d609c0-782b-11e5-9e17-ac55f065607b.png)

**Comment by simonemainardi** [comment]
try to inspect ntopng console output for errors, and possibly post them here. Also please post ntopng parameters.

**Comment by nazhu** [comment]
Not "Flows - Active Flows", but "Hosts - Host:x.x.x.x - Flows - Active Flows".
![screenshot](https://cloud.githubusercontent.com/assets/15213084/10660584/91931afe-78da-11e5-9d4e-ab014cd33133.png)

-------------------------------------------------------------------
My Centos information is:
[ntop@monitor 桌面]$ cat /etc/centos-release 
CentOS release 6.7 (Final)
[ntop@monitor desktop]$ uname -a
Linux monitor 2.6.32-573.7.1.el6.i686 #1 SMP Tue Sep 22 19:05:58 UTC 2015 i686 i686 i386 GNU/Linux
[ntop@monitor desktop]$ ntopng --version
v.2.1.151020 [Community Edition]
GIT rev:   dev:dfc3d2c99d08415df4edae1ead3adcdbe684a037:20151020
[ntop@monitor desktop]$ 

--------------------------------------------------------------------
ntopng conf is:
-G=/var/tmp/ntopng.gid
--local-networks=192.168.0.0/24
-B=tcp
--interface=eth0
--http-port=3000

--------------------------------------------------------------------
ntopng console output info is:
[ntop@monitor desktop]$ sudo /usr/local/bin/ntopng /etc/ntopng/ntopng.conf
22/Oct/2015 16:28:13 [Ntop.cpp:931] Setting local networks to 192.168.0.0/24
22/Oct/2015 16:28:13 [Redis.cpp:105] Successfully connected to redis 127.0.0.1:6379@0
22/Oct/2015 16:28:13 [PF_RINGInterface.cpp:52] Reading packets from PF_RING v.6.1.1 interface eth0...
22/Oct/2015 16:28:13 [Ntop.cpp:1180] Registered interface view eth0 [id: 1]
22/Oct/2015 16:28:13 [PF_RINGInterface.cpp:159] Packet capture filter set to "tcp"
22/Oct/2015 16:28:13 [Ntop.cpp:1150] Registered interface eth0 [id: 0]
22/Oct/2015 16:28:13 [Utils.cpp:304] User changed to nobody
22/Oct/2015 16:28:13 [main.cpp:237] PID stored in file /var/tmp/ntopng.gid
22/Oct/2015 16:28:13 [HTTPserver.cpp:456] HTTPS Disabled: missing SSL certificate /usr/local/share/ntopng/httpdocs/ssl/ntopng-cert.pem
22/Oct/2015 16:28:13 [HTTPserver.cpp:458] Please read https://github.com/ntop/ntopng/blob/dev/doc/README.SSL if you want to enable SSL.
22/Oct/2015 16:28:13 [HTTPserver.cpp:501] Web server dirs [/usr/local/share/ntopng/httpdocs][/usr/local/share/ntopng/scripts]
22/Oct/2015 16:28:13 [HTTPserver.cpp:504] HTTP server listening on port 3000
22/Oct/2015 16:28:13 [main.cpp:290] Working directory: /var/tmp/ntopng
22/Oct/2015 16:28:13 [main.cpp:292] Scripts/HTML pages directory: /usr/local/share/ntopng
22/Oct/2015 16:28:13 [Ntop.cpp:260] Welcome to ntopng i686 v.2.1.151020 - (C) 1998-15 ntop.org
22/Oct/2015 16:28:13 [Ntop.cpp:265] Built on CentOS release 6.7 (Final)
22/Oct/2015 16:28:13 [PeriodicActivities.cpp:53] Started periodic activities loop...
22/Oct/2015 16:28:13 [RuntimePrefs.cpp:32] Dumping alerts into syslog
22/Oct/2015 16:28:13 [Ntop.cpp:544] Adding fe80::20c:29ff:fe5c:c23e/64 as IPv6 local network for eth0
22/Oct/2015 16:28:13 [NetworkInterface.cpp:1450] Started packet polling on interface eth0 [id: 1]...

**Comment by lucaderi** [comment]
Fixed, binary packages are under build.

**Comment by nazhu** [comment]
@lucaderi Yes, it's fixed. :+1:  3q

**Comment by nazhu** [comment]
In the "Fixed for #245", the new version is fix for the Centos6.7, the old version is fix for Debian Jessie.

**Comment by lucaderi** [comment]
I have finally found the real bug that caused this issue. It is fixed. Binary packages are being built
